libs/eo/certificates/src/lib/eo-certificates.component.ts

```diff
@@ -27,20 +28,28 @@ import { EoCertificatesTableComponent } from './eo-certificates-table.component'
     WattCheckboxComponent,
     EoCertificatesTableComponent,
     WattButtonComponent,
+    WattCardComponent,
   ],
   selector: 'eo-certificates',
   standalone: true,
   styles: [],
   template: `
-    <mat-card class="watt-space-stack-l">
-      <h3 class="watt-space-stack-m">This is the beginning</h3>
+    <watt-card class="watt-space-stack-l">
+      <h3 class="watt-space-stack-m">Welcome to the beta test of Energy Origin</h3>
+      <h4 class="watt-space-stack-m">Notice of change:</h4>
       <p class="watt-space-stack-m">
+        Due to changes in the solution regarding certificates, some data has been removed or is
+        planned to be removed. <br />
+        This means that metering points that have been activated for issuance of certificates before
+        <b>August 17th, 2023</b>, must be re-activated. Furthermore, certificates that have been
+        issued before <b>August 23rd, 2023</b>, will be removed. <br />
+        <br />
         This page is based on real data and is working towards the coming solution regarding
         certificates. So it is not just a test, though these data cannot yet be used in a legal
         sense. It will be communicated, when it is out of beta and can be used legally. So you can
         try this without any consequences.
       </p>
-    </mat-card>
+    </watt-card>
```

Comment:
This is duplicated 3 times. Is it possible to create a component for this, which is then used in 3 places? That way we do not have to change the text 3 different places in the future.

Reply:
Fixed, and apparantly it is 4 places. Also when you open up a certificates "details" page. Anyways here is the commit: [5cac50f](https://github.com/Energinet-DataHub/greenforce-frontend/pull/1967/commits/5cac50fbf73b63b17eed960a38c19b8705fbecf6)